Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:gallium-bullseye-slim AS base
+
+ENV NODE_ENV production
+
+WORKDIR /instant
+
+# install curl
+RUN apt-get update; apt-get install -y curl
+
+# install docker engine
+RUN curl -sSL https://get.docker.com/ | sh
+
+# install docker-compose binary
+RUN curl -L "https://github.com/docker/compose/releases/download/1.25.5/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+RUN chmod +x /usr/local/bin/docker-compose
+
+# remove orphan container warning
+ENV COMPOSE_IGNORE_ORPHANS=1
+
+# install node deps
+ADD package.json .
+ADD yarn.lock .
+RUN yarn --prod
```

Comment:
It seems --prod isnt enough with yarn, we might have to include --frozen-lockfile, 
[link](https://stackoverflow.com/questions/49530678/why-will-yarn-install-dev-dependencies-when-i-just-need-the-builds#:~:text=As%20said%20in,npm%20and%20pnpm.)

Reply:
Sounds good 👍🏼

Reply:
done